docs/installing/vagrant.md

```diff
@@ -66,16 +114,20 @@ below, this is `1234`), and issue the `kill` command:
             1234
             $ kill -2 1234
 
-Alternatively, you can shut down the whole VM without deleting it with the
-command <code>vagrant&nbsp;halt</code>
-on the host command line. To start it up again, go to step 2, above &mdash; it
-won't take so long this time, because the files are already in place.
-See [the Vagrant documentation](https://docs.vagrantup.com/v2/)
-for full instructions on using Vagrant.
+* Alternatively, you can shut down the whole VM (without deleting it) with the
+  command <code>vagrant&nbsp;halt</code> (from _outside_ Vagrant, that is, on
```

Comment:
ah, if I had a shiny penny or every time I'd tried to run vagrant commands from _inside_ the vagrant shell \*sigh\*